Add description for scriptlets exceptions (#513)
AdguardTeam/KnowledgeBase: docs/general/ad-filtering/create-own-filters.md

This rule will be applied to `example.org` and subdomains pages and will execute the `abort-on-property-read` scriptlet with the `alert` parameter.

**Exceptions**

Exception rules can disable some scriptlets on particullar domains.

```text
rule = [domains]  "#@%#//scriptlet(" [name] ")"
```

- `domains` — optional, a list of domains where the rule should be applied;

> - `name` — optional, a name of the scriptlet to except from the applying. If not set, all scriptlets will not be applied.

Reviewer: ```suggestion
- `name` — optional, a name of the scriptlet to exclude from applying. If not set, all scriptlets will not be applied.

Author: c77aa1557a01d0bd8e23ab784da0e537d5403433

For example, if you want to disable the rule

```adblock
example.org,example.com#%#//scriptlet("abort-on-property-read", "alert")
```

> for `example.com` domain, the following exception rule could be used:

Reviewer: ```suggestion
for the `example.com` domain, the following exception rule could be used:

Author: c77aa1557a01d0bd8e23ab784da0e537d5403433

**Exceptions**

> Exception rules can disable some scriptlets on particullar domains. The syntax for exception scriptlet rules is similar to normal scriptlet rules but using `#@%#` instead of `#%#`:

Reviewer: ```suggestion
Exception rules can disable some scriptlets on particullar domains. The syntax for exception scriptlet rules is similar to normal scriptlet rules but uses `#@%#` instead of `#%#`:

Scriptlet rules are not supported by AdGuard Content Blocker.

> Full scriptlet exception rules syntax is supported by AdGuard for Windows, Mac, and Android with [CoreLibs] v1.16 or later and AdGuard Browser Extension for Chrome, Firefox, and Edge with [TSUrlFilter] v2.3 or later. Previous versions support only scriptlet exception rules disabling particular scriptlets only.

Reviewer 2: ```suggestion
Full scriptlet exception rules syntax is supported by AdGuard for Windows, Mac, and Android with [CoreLibs] v1.16 or later and AdGuard Browser Extension for Chrome, Firefox, and Edge with [TSUrlFilter] v3.0 or later. Previous versions support only scriptlet exception rules disabling particular scriptlets only.
```

Scriptlet rules are not supported by AdGuard Content Blocker.

> Full scriptlet exception rules syntax is supported by AdGuard for Windows, Mac, and Android with [CoreLibs] v1.16 or later and AdGuard Browser Extension for Chrome, Firefox, and Edge with [TSUrlFilter] v2.3 or later. Previous versions support only scriptlet exception rules disabling particular scriptlets only.

Reviewer: ```suggestion
The full syntax of scriptlet exception rules is supported by AdGuard for Windows, Mac and Android with [CoreLibs] v1.16 or later and AdGuard Browser Extension for Chrome, Firefox and Edge with [TSUrlFilter] v2.3 or later. Previous versions only support scriptlet exception rules that disable specific scriptlets.
